src/frontend/pages/chat/message/message-content.js

```diff
@@ -218,18 +219,18 @@ export class MessageContent extends LitElement {
 
             <div
               class=${
-                this.message.hasBeenDeleted
+                this.message.status === MessageStatuses.deleted
                   ? "timestamp-deleted-container"
                   : "timestamp-edited-container"
               }
             >
               ${when(
-                this.message.hasBeenEdited && !this.message.hasBeenDeleted,
+                this.message.status === MessageStatuses.edited,
                 () => html`<p class="edited">Modificato</p>`,
                 () => html``
               )}
               ${when(
-                this.message.hasBeenDeleted,
```

Comment:
Avrei mantenuto `hasBeenDeleted` come nome, trasformandolo in un metodo che internamente fa `this.message.status === MessageStatuses.deleted`